eagleproject/core/blockchain/rpc.py

```diff
@@ -463,10 +464,15 @@ def staking_durationRewardRate(address, duration, block="latest"):
     return Decimal(int(data["result"], 16))
 
 
-def dripper_available(block="latest"):
+def dripper_available(block="latest", project=OriginTokens.OUSD):
     signature = SIG_DRIPPER_AVAILABLE_FUNDS[:10]
-    data = call(DRIPPER, signature, "", block)
-    return Decimal(int(data["result"], 16)) / E_6
+
+    if project == OriginTokens.OETH:
+        data = call(OETH_DRIPPER, signature, "", block)
+        return Decimal(int(data["result"], 16)) / E_18
+    else:
+        data = call(DRIPPER, signature, "", block)
+        return Decimal(int(data["result"], 16)) / E_6
```

Comment:
OUSD dropper is still returning data in 6 decimals?

Reply:
I believe this is correct no? 
https://etherscan.io/address/0x80c898ae5e56f888365e235ceb8cea3eb726cb58#readProxyContract

![image](https://github.com/OriginProtocol/ousd-analytics/assets/23041116/2ad9ce86-1435-44d3-be6f-583ae9f9a449)
![image](https://github.com/OriginProtocol/ousd-analytics/assets/23041116/eeaac1ad-5c9b-4723-a9cc-01347b430105)

Reply:
Yeah, just checked it against deployed OUSD Dripper and the codebase, seems to be right